build.gradle

```diff
@@ -30,8 +29,8 @@ allprojects {
 
 allprojects {
     plugins.withType(JavaPlugin) {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
this is a major change, projects using Java 8 can no longer use the testcontainer as dependency